Library/AppEnvironmentTests.swift

```diff
@@ -5,6 +5,10 @@ import Prelude
 import XCTest
 
 final class AppEnvironmentTests: XCTestCase {
+  override func setUp() {
+    AppEnvironment.resetStackForUnitTests()
```

Comment:
This keeps any state from the actual application host from leaking into `AppEnvironmentTests`.